### ISSUE TITLE
Test scanning of non-US paper sizes

### ISSUE THREAD

**Issue by mojodna** [fieldpapers/fieldpapers#24]
Since the paper size selector was never enabled on fieldpaper.org (despite the PDF renderer supporting it), I'm suspicious that it may not work quite right.  We need to try printing some atlases with different aspect ratios and scanning them to see if the source paper size is picked up correctly (and that the result is correctly rectified).

It's also possible that this is moot and as long as we have an "image" (PDF) of the source page(s) that's not distorted, the resulting snapshot will be correctly rectified (assuming the OpenCV-based snapshot processor).

In other words, is the paper size only relevant for creating PDFs with the correct size for printing (ignoring printer-imposed margins, which are fickle) (or aspect ratio, really)?

**Comment by sconnelley** [comment]
@mojodna -- sorta related, do you know why the aspect ratio for letter size in the original Fieldpapers was set to 1.5 landscape and .75 portrait, when in theory it should be 1.29 landscape and .77 portrait.  Letter size is 216 mm x 279 mm by the way

**Comment by sconnelley** [comment]
I looked through the original fieldpaper code base and couldn't find any mention as to why this was.  A2 & A3 seemed to be in correct proportion

**Comment by mojodna** [comment]
Not sure, unless it has something to do with the margins (since there's extra space up on top).  The difference between "print points" and "homogenous print points" continues to escape me (referenced in `compose.py` and `decode.py`'s dependencies).

**Comment by mojodna** [comment]
Confirmed that A4 landscape works.

**Comment by mojodna** [comment]
Confirmed that A4 portrait, A3 portrait, and A3 landscape are all good.
